package.json

```diff
@@ -19,6 +19,7 @@
     "lint-fix": "tslint --project . --fix",
     "build": "tsc",
     "test": "jest",
+    "stryker": "stryker run",
```

Comment:
nit: this takes the striker namespace in commands and we can't do `yarn stryker --somethingelse`. Change it to a different name.